x-pack/plugins/osquery/public/live_queries/form/index.tsx

```diff
@@ -501,100 +524,105 @@ const LiveQueryFormComponent: React.FC<LiveQueryFormProps> = ({
     }
   }, [queryType, cleanupLiveQuery, resetField, setValue, clearErrors, defaultValue]);
 
+  const casesOwner = useMemo(() => [SECURITY_SOLUTION_OWNER], []);
+
   return (
     <>
       <FormProvider {...hooksForm}>
-        <EuiFlexGroup direction="column">
-          {queryField && (
-            <EuiFlexItem>
-              <EuiFormRow label="Query type" fullWidth>
-                <EuiFlexGroup gutterSize="m">
-                  <EuiFlexItem>
-                    <StyledEuiCard
-                      layout="horizontal"
-                      title={i18n.translate(
-                        'xpack.osquery.liveQuery.queryForm.singleQueryTypeLabel',
-                        {
-                          defaultMessage: 'Single query',
-                        }
-                      )}
-                      titleSize="xs"
-                      hasBorder
-                      description={i18n.translate(
-                        'xpack.osquery.liveQuery.queryForm.singleQueryTypeDescription',
-                        {
-                          defaultMessage: 'Run a saved query or new one.',
-                        }
-                      )}
-                      selectable={queryCardSelectable}
-                      isDisabled={!canRunSingleQuery}
-                    />
-                  </EuiFlexItem>
-                  <EuiFlexItem>
-                    <StyledEuiCard
-                      layout="horizontal"
-                      title={i18n.translate(
-                        'xpack.osquery.liveQuery.queryForm.packQueryTypeLabel',
-                        {
-                          defaultMessage: 'Pack',
-                        }
-                      )}
-                      titleSize="xs"
-                      hasBorder
-                      description={i18n.translate(
-                        'xpack.osquery.liveQuery.queryForm.packQueryTypeDescription',
-                        {
-                          defaultMessage: 'Run a set of queries in a pack.',
-                        }
-                      )}
-                      selectable={packCardSelectable}
-                      isDisabled={!canRunPacks}
-                    />
-                  </EuiFlexItem>
-                </EuiFlexGroup>
-              </EuiFormRow>
-            </EuiFlexItem>
-          )}
-          {!hideAgentsField && (
-            <EuiFlexItem>
-              <AgentsTableField />
-            </EuiFlexItem>
-          )}
-          {queryType === 'pack' ? (
-            <>
+        <CasesContext owner={casesOwner} permissions={casePermissions}>
```

Comment:
do we need `CasesContext` here? maybe we could move it inside `PackQueriesStatusTable` component?

Reply:
Yes, that should work, thanks 👍

Reply:
Or no, change of plans - we need it for the AddToCaseButton too which is used outside of PackQueriesStatusTable. I'll think about it in the morning ;)

Reply:
my point is that CasesContext should be used as close as it's being used possible, even if we have to do it in 2 places

Reply:
Alright, I moved it both to `ResultsTable` and `PackResultsHeader`

Reply:
After creating a separate component with Context, there was a strange rerending bug in one of the usages. I decided not to change the current solution (working) because of pressure of time. I'll get back to this refactor after FF. I hope it's ok with you Patryk.